Source/Android/jni/ButtonManager.cpp

```diff
@@ -586,13 +586,19 @@ void Init(const std::string& gameId)
       {
         hasbind = true;
         type = BIND_AXIS;
-        sscanf(value.c_str(), "Device '%127[^\']'-Axis %d%c", dev, &bindnum, &modifier);
+        if (std::string::npos != value.find("Device ''"))
```

Comment:
Although it's not mentioned in the Coding Style guide, Yoda conditions are not used in the codebase.

Reply:
There is exactly the same code few lines higher:
      if (std::string::npos != value.find("Axis"))
Should I be consistent with the file or switch arguments?

Reply:
I guess you can keep it as is then.